ha/k8s_setup/ha_setup.py

```diff
@@ -267,7 +268,9 @@ def process(self):
 
             Log.info('Performing event_manager subscription')
             event_manager = EventManager.get_instance()
-            event_manager.subscribe(const.EVENT_COMPONENT, [SubscribeEvent(const.POD_EVENT, ["online", "offline", "failed"])])
+            event_manager.subscribe(const.EVENT_COMPONENT,
+                                    [SubscribeEvent(const.POD_EVENT, ["online", "offline", "failed"], [
```

Comment:
Can you please check why subscription not done for "offline" node event.
Your Test logs : cortx>ha>v2>events>subscribe>hare:["node>server>online", "node>server>failed", "node>data>online", "node>data>failed", "disk>online", "disk>failed"]